src/lib/holocene/textarea.svelte

```diff
@@ -3,12 +3,34 @@
   export let disabled: boolean = false;
   export let placeholder: string = '';
   export let rows: number = 5;
+  export let error: string = null;
+  let hasError = error || $$slots.error;
```

Comment:
Whoa, I dig this.

Reply:
This would have been cool, but apparently you can't wrap slots in if statements within the parent component like I was thinking you could 😞